src/settings/Settings.tsx

```diff
@@ -698,6 +698,10 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         displayName: _td('Send analytics data'),
         default: null,
     },
+    "ftueUseCaseSelection": {
```

Comment:
For naming guidelines on settings:
```suggestion
    "FTUE.useCaseSelection": {
```

will need to be applied elsewhere